web/controllers/user_controller.ex

```diff
@@ -34,6 +35,21 @@ defmodule CodeCorps.UserController do
     end
   end
 
+  def github_connect(conn, code) do
+    current_user = Guardian.Plug.current_resource(conn)
+    with {:ok, user} <- Github.connect(current_user, code)
+    do
+      # {:ok, user}
```

Comment:
Cool, thanks for the feedback! Definitely good to hear I wasn't going crazy trying to figure out how to do it :)